manifests/preinstall.pp

```diff
@@ -0,0 +1,14 @@
+# @summary A short summary of the purpose of this class
+#
+# A description of what this class does
+#
+# @example
+#   include pulpcore::preinstall
+class pulpcore::preinstall {
+
+  include pulpcore::selinux
```

Comment:
I assume this will drop away before its all said and done.

Reply:
I included this based on the following task from the ansible-pulp installer:

https://github.com/pulp/ansible-pulp/blob/86a19f7bf4869cbe55d4850c4d2d53d33e1ee86b/roles/pulp/tasks/install.yml#L49-L53